src/review/logic.py

```diff
@@ -77,6 +81,31 @@ def get_reviewers(article, candidate_queryset, exclude_pks):
     ).annotate(
         rating_average=Subquery(rating_average, output_field=IntegerField()),
     )
+
+    if article.journal.get_setting('general', 'display_past_reviewers'):
```

Comment:
Do we even want this setting? It sounds that when it is off, all we do is remove the column and avoid the initial sorting. To me the name of the setting reads as if you would "hide" the past reviewers if I turned this off.